rust/bin/meta/src/handler.rs

```diff
@@ -15,7 +15,23 @@
 //
 mod meta;
 
-#[derive(Default, Debug)]
+use std::sync::Arc;
+use kv::*;
+
 pub struct Meta {
-    
+    store: Arc<Store>,
+    bucket: Bucket<'static, Raw, Raw>,
```

Comment:
```suggestion
    bucket: Bucket<"static", Raw, Raw>,
```

double quote?

Reply:
This is lifetime annotation in Rust.
https://doc.rust-lang.org/book/ch10-03-lifetime-syntax.html#lifetime-annotations-in-function-signatures